evm/v0.5/contracts/dev/WhitelistedAggregator.sol

```diff
@@ -0,0 +1,80 @@
+pragma solidity 0.5.0;
+
+import "./PrepaidAggregator.sol";
+import "./Whitelisted.sol";
+
+/**
+ * @title Whitelisted Prepaid Aggregator contract
+ * @notice This contract requires addresses to be added to a whitelist
+ * in order to read the answers stored in the PrepaidAggregator contract
+ */
+contract WhitelistedAggregator is PrepaidAggregator, Whitelisted {
+
+  constructor(
+    address _link,
+    uint128 _paymentAmount,
+    uint32 _timeout,
+    uint8 _decimals,
+    bytes32 _description
+  ) public PrepaidAggregator(
+    _link,
+    _paymentAmount,
+    _timeout,
+    _decimals,
+    _description
+  ){}
+
+  /**
+   * @notice get the most recently reported answer
+   * @dev overridden funcion to add the isWhitelisted() modifier
+   */
+  function latestAnswer()
+    external
+    isWhitelisted()
+    view
+    returns (int256)
+  {
+    return rounds[latestRoundId].answer;
```

Comment:
Can we call `super` instead or reimplementing? (Question for all the methods defined here.)